Testscripts/Linux/utils.sh

```diff
@@ -2294,7 +2294,7 @@ function install_sshpass () {
 	which sshpass
 	if [ $? -ne 0 ]; then
 		echo "sshpass not installed\n Installing now..."
-		if [ $DISTRO_NAME == "sles" ] && [[ $DISTRO_VERSION =~ 12 ]]; then
+		if [ $DISTRO_NAME == "sles" ] && [[ $DISTRO_VERSION =~ 12 ]] || [[ $DISTRO_VERSION =~ 15 ]]; then
```

Comment:
Convert to arithmetic comparison? (>= 12)

Reply:
Ok, let me update